.mergify.yml

```diff
@@ -40,14 +44,14 @@ pull_request_rules:
         message: Automatically approving mergify
   - name: automatic squash and merge with success checks and the files matching the regex ^testing/environments/snapshot* are modified.
     conditions:
-      - check-success=beats-ci/pr-merge
       - label=automation
       - files~=^testing/environments/snapshot.*\.yml$
       - "#approved-reviews-by>=1"
+      - check-success=beats-ci/pr-merge
```

Comment:
Can you add this check as used to be, at the beginning of the conditions?

Reply:
moved - check-success=beats-ci/pr-merge to the begin of the list